.github/workflows/test-jinad.yml

```
@@ -0,0 +1,45 @@
+name: Test jinad
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  jinad-test:
+    if: |
+      !startsWith(github.event.head_commit.message, 'chore') &&
+      !startsWith(github.event.head_commit.message, 'build: hotfix') &&
+      !endsWith(github.event.head_commit.message, 'reformatted by jina-dev-bot')
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.7]
+        path: [jinad]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Get jinad repo
+        uses: actions/checkout@v2
+        with:
+          repository: jina-ai/jinad
+          path: jinad
+      - name: Build jina test-pip
+        run: |
+          docker build -f Dockerfiles/pip.Dockerfile -t jinaai/jina:test-pip .
+          docker build --build-arg PIP_TAG="[devel]" -f Dockerfiles/pip.Dockerfile -t jinaai/jina:test-pip .
+      - name: Test jinad
+        run: |
+          cd jinad/${{ matrix.path }}
+          ./tests/integration/jinad/test_simple_distributed/test_integration.sh
```

```suggestion
          cd jina
          ./tests/integration/jinad/test_simple_distributed/test_integration.sh
```
I'm confused about this line. If this is the case, I would expect `test_integration.sh` under the `jinad` repo. However, we put it in the `jina` repo
```suggestion
          cd jinad/${{ matrix.path }}
          ./tests/integration/jinad/test_simple_distributed/test_integration.sh
```

---

I would keep it like this now, and in another PR add a tests.sh in the jinad root an there put the logic to tesr every subfolder. 

And I woulf also kwep these tests in core repo to see every PR.

---

Ah well it is true that the real action has not been tested, I think @rutujasurve94 u can commit these suggesions

---

Done, doing `cd jina` instead of `matrix path` for jinad